sdl-debezium/src/main/scala/io/smartdatalake/workflow/connection/DebeziumConnection.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2024 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.smartdatalake.workflow.connection
+import com.typesafe.config.Config
+import io.debezium.connector.db2.Db2Connector
+import shaded.io.debezium.connector.mariadb.MariaDbConnector
+import io.debezium.connector.mongodb.MongoDbConnector
+import shaded.io.debezium.connector.mysql.MySqlConnector
+import io.debezium.connector.oracle.OracleConnector
+import io.debezium.connector.postgresql.PostgresConnector
+import io.debezium.connector.spanner.SpannerConnector
+import io.debezium.connector.sqlserver.SqlServerConnector
+import io.debezium.connector.vitess.VitessConnector
+import io.smartdatalake.config.SdlConfigObject.ConnectionId
+import io.smartdatalake.config.{FromConfigFactory, InstanceRegistry}
+import io.smartdatalake.workflow.connection.authMode.{AuthMode, BasicAuthMode}
+
+/**
+ * Connection information for debezium connection
+ *
+ * @param id unique id of this connection
+ * @param dbEngine what database engine to use, currently supported engines: mysql, postgresql, oracle, mariadb, mongodb, sqlserver, db2, vitess, spanner
+ * @param hostname database server
+ * @param db database to read data from
+ * @param port
+ * @param authMode authentication information: for now BasicAuthMode is supported.
+ * @param metadata optional connection metadata
+ */
```

Comment:
Can you add an example config in the doc? Something like this?
https://github.com/smart-data-lake/smart-data-lake/blob/master-spark3/sdl-core/src/main/scala/io/smartdatalake/workflow/action/generic/transformer/ColumnsTransformer.scala#L43